python/adelphi/adelphi/gemini.py

```diff
@@ -74,11 +77,8 @@ def export_gemini_schema(keyspaces_metadata, options):
 
         data["tables"].append(table_data)
 
-    return data
-
+    return metadata_str + "\n\n" + json.dumps(data, indent=4)
 
-def format_gemini_schema(data):
-    return json.dumps(data, indent=4)
```

Comment:
The export/format distinction wasn't really buying us much.  We can always reintroduce it if we find out it's useful in some way.